qa/SidechainTestFramework/account/smart_contract_resources/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "smart_contract_resources",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "engines": {
+    "npm": ">=8.1.2",
+    "node": ">=16.13.2"
```

Comment:
Let's add the requirements info to the `README.md` file.
I want to see there that we need to install `npm`, `node` and `yarn` of the given versions (`>=...`).
Then python requirements specified in `account/requirements.txt`